marxs/source/labSource.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
+from astropy.table import Table, Column
+
+from .source import Source
+
+
+class FarLabConstantPointSource(Source):
+	'''Simple in-lab source for testing purposes
+
```

Comment:
You should describe here what the format for all the inputs is, e.g. is position a  numpy array that describes 3d vector of x,y,z? A 4d vector in homogeneos coordintes? A 4*4 matrix like the pos4d for the other elements? Or maybe just a number for the x-value and you assume that it's on the x-axis with y=z=0?

The preferred format is:
```
Parameters
----------------
position: np.array
    4d homogeneous vecotr that describes the center of the aperture (or some other text).
```

See some of the other classes for examples.